airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -359,7 +358,8 @@ public void testCreateDestination() throws ApiException {
     assertEquals(name, createdDestination.getName());
     assertEquals(destinationDefId, createdDestination.getDestinationDefinitionId());
     assertEquals(workspaceId, createdDestination.getWorkspaceId());
-    assertEquals(getDestinationDbConfigWithHiddenPassword(), createdDestination.getConnectionConfiguration());
+    assertEquals(getDestinationDbConfigWithHiddenPassword(),
```

Comment:
why are there so many blank space changes in this file?

Reply:
Jared, did you change your line wrap?